cli/internal/jsonmessage/display.go

```diff
@@ -0,0 +1,32 @@
+package jsonmessage
```

Comment:
If we could name this package `jsonstream` it could help disambiguate with the `jsonmessage` package, and we could name the function `Display()` as it would be called as  `jsonstream.Display()`